content/en/dynamic_instrumentation/_index.md

```diff
@@ -39,20 +43,41 @@ Dynamic Instrumentation requires the following:
 - [Unified Service Tagging][5] tags `service`, `env`, and `version` are applied to your deployment.
 - Optionally, [Source Code Integration][6] is set up for your service.
 
+**Note**: `debugger_read` and `debugger_write` permissions required to access Dynamic Instrumentation. While Datadog Administrator have those permissions by default, for more information about default and custom roles and assign roles to users, see [Role Based Access Control][11].
```

Comment:
```suggestion
**Note**: `debugger_read` and `debugger_write` permissions are required to access the Dynamic Instrumentation page. For more information about roles and on how to assign roles to users, see [Role Based Access Control][11].
```
Also, should we rename the roles? 😅

Reply:
I was thinking the same. but we decided to keep those when we rename the product - will sync with Evgeny to see if that changed.